joomla-projects/custom-fields
# Missing quote mark throws PHP notice

Pull Request for Issue #175 .
### Summary of Changes

PHP notice "Use of undefined constant fullname - assumed 'fullname' in administrator\components\com_fields\helpers\internal.php on line 109" is visible in field view because of the missing quote marks.
### Testing Instructions

Create a new field, notice is visible.
Add quote mark around word 'fullname' on line 109, save.
Again try to create a new field, notice is gone.
### Documentation Changes Required
